src/index.js

```diff
@@ -0,0 +1,6 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import './index.scss';
+import App from './App';
+
+ReactDOM.render(<App />, document.querySelector('#root'));
```

Comment:
let add hot reload here to make your life easier

https://medium.com/@brianhan/hot-reloading-cra-without-eject-b54af352c642